src/lib.rs.in

```diff
@@ -674,7 +688,7 @@ class IronSdk {
     self_type IronOxide;
     private constructor = empty;
     static_method user_verify(_:&str) -> Result<Option<UserVerifyResult>, String>; alias userVerify;
-    static_method user_create(_:&str, _:&str) -> Result<UserCreateKeyPair, String>; alias userCreate;
+    static_method user_create(_:&str, _:&str, _:&UserCreateOpts) -> Result<UserCreateKeyPair, String>; alias userCreate;
```

Comment:
It would be nice if the params had names and we had nice comments on these 2 like we do on other methods.

Reply:
I named the parameters and brought the comments from ironoxide here as well